packages/ckeditor5-dev-env/tests/translations/collect-utils.js

```diff
@@ -16,16 +16,42 @@ const glob = require( 'glob' );
 const fs = require( 'fs-extra' );
 
 describe( 'collect-utils', () => {
-	let sandbox;
+	const sandbox = sinon.sandbox.create();
```

Comment:
Ok. I can move it inside `beforeEach`.
But I have one advantage of this approach - my IDE recognizes easily the type of the `sandbox` variable. Otherwise it becomes an `any` type.